kensu/pyspark/spark_connector.py

```diff
@@ -801,8 +801,14 @@ def extract_prop(name, tpe, value):
             if (maybe_ds_path_sanitizer_search is not None) and (maybe_ds_path_sanitizer_replace is not None):
                 add_ds_path_sanitizer(spark_session, maybe_ds_path_sanitizer_search, maybe_ds_path_sanitizer_replace)
 
-            w = jvm.io.kensu.sparkcollector.KensuSparkCollector.KensuSparkSession(spark_session._jsparkSession)
-            w.track(ingestion_url, jvm.scala.Option.empty(), properties.toSeq())
+            if is_databricks():
+                print('Detected Databricks Notebook - initializing Kensu via DatabricksCollector.track()')
+                w = ref_scala_object(jvm, 'io.kensu.sparkcollector.environments.DatabricksCollector')
```

Comment:
unfortuantely stdout is lost  from cell output (including the config printing & connection check -> visible only in cluster logs) if initializing this way with `init_kensu_spark()`

what works in Databricks: